backend/app/cmd/server.go

```diff
@@ -133,10 +133,11 @@ type NotifyGroup struct {
 
 // SSLGroup defines options group for server ssl params
 type SSLGroup struct {
-	Type string `long:"type" env:"TYPE" description:"ssl (auto)support" choice:"none" choice:"static" choice:"auto" default:"none"`
-	Port int    `long:"port" env:"PORT" description:"port number for https server" default:"8443"`
-	Cert string `long:"cert" env:"CERT" description:"path to cert.pem file"`
-	Key  string `long:"key" env:"KEY" description:"path to key.pem file"`
+	Type  string `long:"type" env:"TYPE" description:"ssl (auto)support" choice:"none" choice:"static" choice:"auto" default:"none"`
+	Port  int    `long:"port" env:"PORT" description:"port number for https server" default:"8443"`
+	Cert  string `long:"cert" env:"CERT" description:"path to cert.pem file"`
+	Key   string `long:"key" env:"KEY" description:"path to key.pem file"`
+	Cache string `long:"cache" env:"CACHE" description:"dir where certificates will be stored by autocert manager" default:"./var/certs"`
```

Comment:
parameter named "cache" can be confusing, as it conflict with the current cache (means smth else). It makes sense to name it "location", but even better - get rid of it completely and just use existing Cert and Key

Reply:
I agree, but acme library expects path to directory where cert and key are stored, not paths to cert and key files. So `static` mode expects Cert and Key params, but `auto` mode expect only one directory param. So don't understand how to union them in common maybe rename it to "location"?

Reply:
two possible solutions:
- get the directory from cert or key
- just name this param "acme_location"

Probably the second less confusing